### ISSUE TITLE
Cron Task is OK but reported as failed only after 10 minutes

### ISSUE THREAD

**Issue by Tangeek42** [nextcloud/server#19314]
Hi,

I haven't seen it reported yet, but if it has do not hesitate to close this and/or merge with another ticket. Also, I have a tendency to write a little too much, so I apologize already if I do, I'm just trying to be thorough.

### Steps to reproduce
1. Execute the cron task, either automatically or manually
2. Check that the page reports the task as OK
3. Wait 10 minutes
4. Check again, it reports as "something went wrong on the last task 10 minutes ago"

### Expected behaviour
The task should always report OK, regardless of "how long was it ago".

### Actual behaviour
The cron task seems to be executing well, but strangely enough, it is reported as failed *only* when the last task is over 10 minutes ago. If I check the page before 10 minutes, the task is reported as OK.
I think the task itself is actually executing well because I had a few notifications. But I can't be sure, there's nothing showing in the logs and even after redirecting the output in cron, I got nothing. The php-fpm logs are empty as well.

### Server configuration

**Operating system:**
Linux, Debian 9.11

**Web server:**
Apache 2.4.25 (Debian package)

**Database:**
MySQL 15.1 (Debian package)

**PHP version:**
7.3.14 (compiled manually, but the cron calls the correct one of course). In the overview page all the tests are reported OK.

**Nextcloud version:** (see Nextcloud admin page)
17.0.3

**Updated from an older Nextcloud/ownCloud or fresh install:**
Upgraded from a 15.something (I can't remember the minor and I don't have any notes or screens unfortunately)

**Where did you install Nextcloud from:**
From nextcloud.com, the official tar.gz, and upgraded via the web updater

**Signing status:**
```No errors have been found.```

**List of activated apps:**
```Enabled:
  - accessibility: 1.3.0
  - activity: 2.10.1
  - audioplayer: 2.9.0
  - bruteforcesettings: 1.5.0
  - cloud_federation_api: 1.0.0
  - comments: 1.7.0
  - dav: 1.13.0
  - drawio: 0.9.4
  - federatedfilesharing: 1.7.0
  - federation: 1.7.0
  - files: 1.12.0
  - files_antivirus: 2.2.1
  - files_pdfviewer: 1.6.0
  - files_rightclick: 0.15.1
  - files_sharing: 1.9.0
  - files_trashbin: 1.7.0
  - files_versions: 1.10.0
  - files_videoplayer: 1.6.0
  - firstrunwizard: 2.6.0
  - gallery: 18.4.0
  - logreader: 2.2.0
  - lookup_server_connector: 1.5.0
  - nextcloud_announcements: 1.6.0
  - notifications: 2.5.0
  - oauth2: 1.5.0
  - password_policy: 1.7.0
  - privacy: 1.1.0
  - provisioning_api: 1.7.0
  - ransomware_detection: 0.6.0
  - recommendations: 0.5.0
  - richdocuments: 3.5.2
  - serverinfo: 1.7.0
  - sharebymail: 1.7.0
  - support: 1.0.1
  - survey_client: 1.5.0
  - systemtags: 1.7.0
  - text: 1.1.1
  - theming: 1.8.0
  - twofactor_backupcodes: 1.6.0
  - updatenotification: 1.7.0
  - viewer: 1.2.0
  - workflowengine: 1.7.0
Disabled:
  - admin_audit
  - encryption
  - files_external
  - user_ldap
```
**Nextcloud configuration:**
```
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "nextcloud.domain.com",
            "partage.domain.com",
            "10.0.42.231"
        ],
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/nextcloud.domain.com",
        "dbtype": "mysql",
        "version": "17.0.3.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_smtpdebug": true,
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "PLAIN",
        "mail_smtpsecure": "tls",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "theme": "",
        "loglevel": 2,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_sendmailmode": "smtp",
        "updater.release.channel": "stable",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "updater.secret": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "htaccess.RewriteBase": "\/"
    }
}
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No.

**Are you using encryption:** yes/no
No.

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
No.

### Client configuration
**Browser:**
Firefox 72
**Operating system:**
Linux

#### Web server error log
Those errors don't seem to be related according to the timestamps. But here you go.
```
[Thu Feb 06 09:38:07.434467 2020] [authz_core:error] [pid 26435] [client 10.0.42.201:45736] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
[Thu Feb 06 09:49:38.697703 2020] [authz_core:error] [pid 26437] [client 10.0.42.201:46022] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
[Thu Feb 06 09:52:22.902896 2020] [authz_core:error] [pid 23666] [client 10.0.42.201:46130] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
[Thu Feb 06 09:54:49.239225 2020] [authz_core:error] [pid 26340] [client 10.0.42.201:46194] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
[Thu Feb 06 10:09:31.177779 2020] [authz_core:error] [pid 27400] [client 10.0.42.201:46594] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
[Thu Feb 06 10:09:42.397630 2020] [authz_core:error] [pid 27324] [client 10.0.42.201:46610] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
[Thu Feb 06 10:10:08.842484 2020] [authz_core:error] [pid 23664] [client 10.0.42.201:46626] AH01630: client denied by server configuration: /usr/local/etc/nextcloud/data/.ocdata
```

#### Nextcloud log (data/nextcloud.log)
I just have this line repeating a few trillion times. It's just because I haven't installed Clam yet.

```
{"reqId":"cnqkPdQji3ymGD9v8ug8","level":4,"time":"2020-02-06T09:19:26+00:00","remoteAddr":"","user":"--","app":"files_antivirus","method":"","url":"--","message":"OCA\\Files_Antivirus\\BackgroundJob\\BackgroundScanner::run, exception: The antivirus executable could not be found at \/usr\/bin\/clamscan","userAgent":"--","version":"17.0.3.1"}
```

#### Browser log
Since it's all server side I don't think it matters. But if you think so please ask, I'll do it.

Here are three screenshots illustrating the bug. I swear on my honor I didn't do anything between them except wait. ^^ 
![8m](https://user-images.githubusercontent.com/11335279/73923134-a65e1a00-48ca-11ea-962b-fa8d36220e86.png)
![9m](https://user-images.githubusercontent.com/11335279/73923139-a8c07400-48ca-11ea-8731-c38d2cae2a2c.png)
![10m](https://user-images.githubusercontent.com/11335279/73923144-a9f1a100-48ca-11ea-8417-0d6ce0b4801b.png)

EDIT : Sorry, I seem to have a bit of trouble with the details and summary tags. I took it directly from the template, though. ?_?
EDIT2 : I can't seem to make the details tag work properly. I'll leave it like that for now, at least it's readable.

**Comment by kesselb** [comment]
How often do you run `cron.php`?

**Comment by Tangeek42** [comment]
Every 15 minutes. Each time the first ten in between are OK, then it goes red for the remaining five. Then another task is executed, and so on.

Here's the crontab just in case I'm missing something. I've tried logging it but there's no output. (Don(t know if that's normal)

```*/15 * * * * /usr/local/bin/php -f /usr/local/etc/nextcloud/cron.php > /usr/local/etc/nextcloud/LOGCRON 2>&1```

**Comment by kesselb** [comment]
![image](https://user-images.githubusercontent.com/3902676/73930136-56855000-48d6-11ea-8d16-976ab0ae110c.png)

I don't speak french. What does the sentence below cron says?

**Comment by Tangeek42** [comment]
My bad, didn't think of that.
It says to "use the cron system to call the cron.php file every 5 minutes. That file must be executed with the user www-data".

Every. 5. Minutes. I'm dumb. Sorry. I could've sworn it was 15 before, but maybe I was just plain wrong the entire time. >.>

I'll set it to 5. Sorry again. t_t

**Comment by kesselb** [comment]
You are correct. It was 15 minutes for 16 and before.
